### DISCUSSION TITLE
Override config of sourced component

### DISCUSSION THREAD

**Discussion by kinghuang** [explosion/spaCy#9640] [Help: Coding & Implementations] [answered]
Is it possible to override a sourced component's config in a config file? I'm trying set `overwrite = false` on a sourced `entity_linker` component like so.

```ini
[components.entity_linker]
source = ${vars.base_model}
component = "entity_linker"
overwrite = false
```

But, the `overwrite` flag remains at True in the assembled model.

**Comment by polm** [comment, marked as answer]
This is a current limitation of sourced components - they copy the config of the original component exactly, and other arguments in the sourcing config are not used. Fixing this so you can actually override options is on our to-do list.

**Reply by kinghuang** [reply]
Ok. Good to know it's on the to-do list! :)
